pkg/kubelet/lifecycle/handlers.go

```diff
@@ -61,7 +61,7 @@ func (hr *HandlerRunner) Run(containerID kubecontainer.ContainerID, pod *api.Pod
 			msg    string
 		)
 		output := ioutils.WriteCloserWrapper(&buffer)
-		err := hr.commandRunner.ExecInContainer(containerID, handler.Exec.Command, nil, output, output, false, nil)
+		err := hr.commandRunner.ExecInContainer(containerID, handler.Exec.Command, nil, output, output, false, nil, 0)
```

Comment:
I feel like we should probably have a timeout on this, but it's outside the scope of this PR. Maybe add a TODO (with an associated issue).